### DISCUSSION TITLE
Eender 3 V2 - 4.2.7 - Sprite Pro - CR Touch - Dual Z Axis - Is there a compatable firmare ANYWHERE or have I gone too far?

### DISCUSSION THREAD

**Discussion by floydian68** [mriscoc/Ender3V2S1#1202] [General]
I feel like I have gone too far with my upgrades and now there is no longer firmware that is compiled for my printer.
I am not knowledgeable enough to compile my own so my printer is sitting dormant.

Any ideas or recommendations?

**Comment by mriscoc** [comment]
Try with the Ender3V2 427 BLTUBL T13 https://github.com/mriscoc/Special_Configurations/releases/tag/T13

**Comment by teatimewithty** [comment]
okay so i have the same issue and i can't get the nozzle or CR touch to stay on print bed... any solutions?
